podvm/files/etc/kata-opa/disallow-all-except-setpolicy.rego

```diff
@@ -0,0 +1,39 @@
+package agent_policy
+
+default AddARPNeighborsRequest := false
+default AddSwapRequest := false
+default CloseStdinRequest := false
+default CopyFileRequest := false
+default CreateContainerRequest := false
+default CreateSandboxRequest := false
+default DestroySandboxRequest := false
+default GetMetricsRequest := false
+default GetOOMEventRequest := false
+default GuestDetailsRequest := false
+default ListInterfacesRequest := false
+default ListRoutesRequest := false
+default MemHotplugByProbeRequest := false
+default OnlineCPUMemRequest := false
+default PauseContainerRequest := false
+default PullImageRequest := false
+default ReadStreamRequest := false
+default RemoveContainerRequest := false
+default RemoveStaleVirtiofsShareMountsRequest := false
+default ReseedRandomDevRequest := false
+default ResumeContainerRequest := false
+default SetGuestDateTimeRequest := false
+default SignalProcessRequest := false
+default StartContainerRequest := false
+default StartTracingRequest := false
+default StatsContainerRequest := false
+default StopTracingRequest := false
+default TtyWinResizeRequest := false
+default UpdateContainerRequest := false
+default UpdateEphemeralMountsRequest := false
+default UpdateInterfaceRequest := false
+default UpdateRoutesRequest := false
+default WaitProcessRequest := false
+default WriteStreamRequest := false
+default ExecProcessRequest := false
+
+default SetPolicyRequest := true
```

Comment:
does it make sense to generate those from protobuf files? If the agent api evolves we might forget to copy a fn. https://github.com/kata-containers/kata-containers/blob/ec30d5a9a8912ade2b982a8f952ce4e714651ee3/src/libs/protocols/protos/agent.proto#L21

Reply:
I agree. Probably can be taken up in subsequent PR since I think it's  better to make this change in main and then copy the same logic here.